helm/ph-ee-engine-oaf/values.yaml

```diff
@@ -155,12 +155,23 @@ mpesa:
   ingress:
     enabled: true
     annotations: {}
+  safaricom:
+    auth_host: ""
+  secret:
+    name: ""
+    key:
+      auth_client_key: ""
 
 # pesa core connector
 roster_connector: 
   enabled: true
   SPRING_PROFILES_ACTIVE: "med"
   image: ""
+  ams:
```

Comment:
The linter values commit is adding this new code and I don't think this indentation is right.

Reply:
The linter is expecting this indication. I didn't do a dry run for the changes yet.